osu.Game/Screens/OnlinePlay/Multiplayer/Match/Playlist/MultiplayerPlaylistTabControl.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Screens.OnlinePlay.Multiplayer.Match.Playlist
+{
+    public class MultiplayerPlaylistTabControl : OsuTabControl<MultiplayerPlaylistDisplayMode>
+    {
+        public Bindable<int> queueListCount = new Bindable<int>(0);
+
+        public MultiplayerPlaylistTabControl()
+        {
+        }
+
+        protected override TabItem<MultiplayerPlaylistDisplayMode> CreateTabItem(MultiplayerPlaylistDisplayMode value)
+        {
+            if (value == MultiplayerPlaylistDisplayMode.Queue)
+                return new QueueTabItem(value, queueListCount);
+            return new OsuTabItem(value);
+        }
+
+        private class QueueTabItem : OsuTabItem
+        {
+            public QueueTabItem(MultiplayerPlaylistDisplayMode value, Bindable<int> queueListCount)
+                : base(value)
+            {
+                queueListCount.BindValueChanged(
```

Comment:
In all `Drawable`s, never listen to value changes from a bindable that's not stored locally on the drawable itself (that could lead to event leaks), define a local bindable and bind that to the extenral source to listen from it instead.

In other words, this is the recommended pattern (including using `LoadComplete` instead of constructor):
```csharp
new QueueTabItem { QueueListCount = { BindTarget = QueueListCount } }

...

public class QueueTabItem
{
    public readonly IBindable<int> QueueListCount = new BindableInt();

    protected override void LoadComplete()
    {
        base.LoadComplete();
        QueueListCount.BindValueChanged(v => Text.Text = $"Queue ({v.NewValue})", true);
    }
}
```